gcloud/orb.yaml

```diff
@@ -88,6 +88,50 @@ commands:
       - run: ln -s /root/google-cloud-sdk/bin/gsutil /usr/bin/gsutil
       - run: ln -s /root/google-cloud-sdk/bin/docker-credential-gcloud /usr/bin/docker-credential-gcloud
 
+  update-autoscaler:
```

Comment:
Looks like this command is sufficiently generalized that we can avoid any of the `autoscaler` naming/description -- looks like you've got a `patch-configmap-then-force-new-deployment` command... perhaps that should actually be two commands? `gcloud/patch-deployment` and `gcloud/force-deploy-gke` might be a nice, composeable level of granularity?

Reply:
I like the idea of generalizing this now that you mention it. What I don't think works though is generalizing patching the configmap. This only updates one entry in the configmap and not the whole configmap but perhaps that should just be reflected in the naming of the command.

I like `gcloud/force-deploy-gke` although with some naming changes since it's more of a re-deploy since it's not changing anything meaningful.